specification/devcenter/DevCenter/client.tsp

```diff
@@ -0,0 +1,246 @@
+import "./Environments/routes.tsp";
+import "./DevCenter/routes.tsp";
+import "./DevBox/routes.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.Core;
+using TypeSpec.Versioning;
+using DevCenterService;
+using Azure.ClientGenerator.Core;
+
+@useDependency(APIVersions.v2023_04_01)
+namespace SdkCustomizations;
+
+@client({
+  name: "DevCenterClient",
+  service: DevCenterService,
+})
+interface DevCenterClientOperations {
+  listProjects is DevCenterService.DevCenterOperations.listProjects;
+  getProject is DevCenterService.DevCenterOperations.getProject;
+}
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "detailed suppress description for each operation can be found in routes.tsp"
+@client({
+  name: "DevBoxesClient",
+  service: DevCenterService,
+})
+interface DevBoxesClientOperations {
+  listPools is DevCenterService.DevBoxesOperations.listPools;
+  getPool is DevCenterService.DevBoxesOperations.getPool;
+  listSchedules is DevCenterService.DevBoxesOperations.listSchedules;
+  getSchedule is DevCenterService.DevBoxesOperations.getSchedule;
+  listAllDevBoxes is DevCenterService.DevBoxesDevCenter.listAllDevBoxes;
+  listAllDevBoxesByUser is DevCenterService.DevBoxesDevCenter.listAllDevBoxesByUser;
+  listDevBoxes is DevCenterService.DevBoxesOperations.listDevBoxes;
+  getDevBox is DevCenterService.DevBoxesOperations.getDevBox;
+  @convenientAPI(false)
```

Comment:
would be good to add a comment linking to the tracking issue for why we are turning off convenientAPI generation